tests/core/test_views.py

```diff
@@ -122,3 +122,18 @@ def test_coc_redirect(client):
         resp = client.get(old_url)
         assert resp.status_code == 301
         assert resp['Location'] == new_url
+
+
+def test_crowdfunding_donors(client, visible_donors, hidden_donors):
+    # Access crowdfunding donors page
+    resp = client.get(reverse('core:crowdfunding-donors'))
+    assert resp.status_code == 200
+
+    # Check if it returns list of donors in response
+    donors = visible_donors
+    assert donors in resp.context
+
+    # Only the visible donors
+    donor_ids = set(donor.pk for donor in resp.context['donors'])
```

Comment:
I believe in context we get `donor_list` instead of `donors`. We should use correct name.